templates/zerver/help/send-someone-a-private-message.md

```diff
@@ -0,0 +1,31 @@
+# Send someone a private message
+
+Instead of sending a new private message through the compose box, you can
+send a private message to someone through the user list on the right
+sidebar.
+
+1. In the user list on the right sidebar, hover over the name of the user
+you want to privately message to reveal a down chevron (<i
+class="icon-vector-chevron-down"></i>) icon to the right of the user's name.
+
+    !!! tip ""
+        You can search for a specific user by entering the user's name in
+        the **Search users** input.
+
+2. Click on the {!down-chevron.md!} Select the
+**Compose a message to (user's name)** option from the actions dropdown.
```

Comment:
Clicking on the user's name is sorta a faster process; do we want to document that instead?